src/Executor/Executor.php

```diff
@@ -245,12 +268,9 @@ private static function executeFieldsSerially(ExecutionContext $exeContext, Obje
      */
     private static function executeFields(ExecutionContext $exeContext, ObjectType $parentType, $source, $path, $fields)
     {
-        // Native PHP doesn't support promises.
-        // Custom executor should be built for platforms like ReactPHP
         return self::executeFieldsSerially($exeContext, $parentType, $source, $path, $fields);
```

Comment:
I afraid this breaks spec compatibility in section about [field execution order](http://facebook.github.io/graphql/#sec-Normal-and-Serial-Execution). 

The basic idea is that root-level fields of mutations must be executed sequentially, but with this change order of their execution is not guaranteed (since if some field returns `Promise` we delay it until other fields complete).

Previously we always had sequential execution but with promises it is not the case anymore. So `executeFields` and `executeFieldsSerially` must behave differently.

Can you change it to behaive according to spec?

Alternative option would be to disallow Promises returned from root-level fields (for mutations only), but such limitation seems unnecessary.